frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -934,6 +982,14 @@ function ReaderHighlight:onShowHighlightDialog(page, index, is_auto_text)
                     self.edit_highlight_dialog = nil
                 end,
             },
+            {
+                text = _("Highlight color"),
```

Comment:
I'm afraid the width of the button table is not enough to get this new button without truncation on 6".
Maybe just "Color" would fit.

But it definitely should not appear on b/w devices (same concerns the menu item).

Reply:
Word wrap and auto scaling actually meant everything fit in the box. But I think you are right, being one word will look more consistent.